core-web/libs/sdk/angular/README.md

```diff
@@ -1,229 +1,599 @@
+
+**Usage**:
+
+```typescript
+import { DotCMSPageAsset } from '@dotcms/types';
+import { DotCMSLayoutBody } from '@dotcms/angular/next';
+
+@Component({
+    template: `
+        <dotcms-layout-body
+            [page]="pageAsset()"
+            [components]="components()"
+            mode="development"
+        />
+    `
+})
+export class PageComponent {
+    components = signal<DotCMSPageComponent>({
+        Blog: import('./blog.component').then(c => c.BlogComponent)
+    });
+    pageAsset = signal<DotCMSPageAsset | null>(null);
+
+    constructor(private readonly client: DotCmsClient) {}
+
+    ngOnInit() {
+        this.client.page
+            .get({ url: '/my-page' })
+            .then(({ pageAsset }) => {
+                this.pageAsset.set(pageAsset);
+            });
+    }
+}
 ```
 
-Add this configuration to `ApplicationConfig` in your Angular app.
+**Component Mapping**:
+
+* The key `Blog` is the content type variable in dotCMS
+* The value `BlogComponent` is the Angular component that renders the blog post
+
+**Mode**:
+
+* `development` mode is used for development and debugging
+* `production` mode is used for production
+
+
+**Editor Integration**:
+
+* Automatically detects if it's inside dotCMS UVE
+* Works with `DotCMSEditablePageService` for real-time updates
+
+**Common Issues**:
+
+* Missing component mappings cause unrendered containers
+* Editing layout structure manually may break UVE sync
+* Modifying the layout structure directly may break the editor's ability to track changes
+
+
+#### DotCMSEditableText
+
+**Overview**: Component for inline editing of text fields in dotCMS, supporting plain text, text area, and WYSIWYG fields.
+
+**Inputs**:
+
+| Input        | Type                | Required | Description                                       |
+|--------------|---------------------|----------|---------------------------------------------------|
+| `contentlet` | `DotCMSContentlet`  | ✅        | The contentlet containing the editable field      |
+| `fieldName`  | `string`            | ✅        | Name of the field to edit                         |
+| `mode`       | `'plain' \| 'full'` | ❌        | Editor mode (default: 'plain')                    |
+| `format`     | `'text' \| 'html'`  | ❌        | Output format (default: 'text')                   |
+
+**Usage**:
 
-`src/app/app.config.ts`
 ```typescript
-import { InjectionToken } from '@angular/core';
-import { ClientConfig, DotCmsClient } from '@dotcms/client';
+import { DotCMSBasicContentlet } from '@dotcms/types';
+import { DotCMSEditableText } from '@dotcms/angular/next';
+
+@Component({
+    selector: 'app-your-component',
+    imports: [DotCMSShowWhen],
+    template: `
+        <dotcms-editable-text
+            [contentlet]="contentlet"
+            fieldName="title"
+            mode="plain"
+        />
+    `
+})
+export class YourComponent {
+    @Input() contentlet: DotCMSBasicContentlet;
+}
+```
```

Comment:
Make an example of using this inside a contentlet component, like a simple banner or something. So user can see it in action.

Reply:
Done!